src/components/breadcrumb/BreadcrumbNavigation.vue

```diff
@@ -0,0 +1,95 @@
+<script>
+import { defineComponent } from 'vue';
+
+import ArrowheadRight from '@/assets/icons/arrowhead-right.svg';
+
+export default defineComponent({
+  components: {
+    ArrowheadRight,
+  },
+  props: {
+    secondItem: {
+      type: String,
+      default: '',
+    },
+  },
+  methods: {
+    capitalizeFirstCharacter(string) {
+      return string[0].toUpperCase() + string.slice(1);
+    },
+    formatRouteName(name) {
+      const routeName = name.split('-').join(' ');
+      return this.capitalizeFirstCharacter(routeName);
+    },
```

Comment:
You could move this one to be computed as we always use the `firstRouteElement` as the argument.
So we could have a `formattedRouteName` computed which would just use `this.firstRouteElement` in the logic inside 😄